x-pack/plugins/security_solution/common/utils/path_placeholder.test.ts

```diff
@@ -6,16 +6,12 @@
  */
 
 import { getPlaceholderTextByOSType, getPlaceholderText } from './path_placeholder';
-import {
-  ConditionEntryField,
-  OperatingSystem,
-  TrustedAppEntryTypes,
-} from '@kbn/securitysolution-utils';
+import { ConditionEntryField, OperatingSystem, EntryTypes } from '@kbn/securitysolution-utils';
 
 const trustedAppEntry = {
   os: OperatingSystem.LINUX,
   field: ConditionEntryField.HASH,
-  type: 'match' as TrustedAppEntryTypes,
+  type: 'match' as EntryTypes,
```

Comment:
If the condition entry type for trusted apps is `'match' | 'wildcard'` this should have the same type casting.
In case someone changes this `'match'` for `'match_any'` the `EntryTypes` type will allow it and it's not correct for trusted apps.